ledger/src/snapshot_utils.rs

```diff
@@ -30,7 +31,13 @@ pub const TAR_SNAPSHOTS_DIR: &str = "snapshots";
 pub const TAR_ACCOUNTS_DIR: &str = "accounts";
 pub const TAR_VERSION_FILE: &str = "version";
 
-pub const SNAPSHOT_VERSION: &str = "1.0.0";
+pub const SNAPSHOT_VERSION_1_0: &str = "1.0.0";
+pub const SNAPSHOT_VERSION_1_1: &str = "1.0.1";
+
+pub enum BankVersions {
+    Bank1_0(Bank1_0),
+    Bank1_1(Bank),
```

Comment:
Bank1_1 we can just call `Current`, and it only gets a version when it's an older bank that we care about